### PR TITLE
BREAKING CHANGE - Uno.Themes into main

### PR DESCRIPTION
**Pull request by pictos**
﻿GitHub Issue: #1289

Missing docs

## PR Type

What kind of change does this PR introduce?
- Feature

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Converters/FirstCharacterConverter.cs
Comment (Youssef1313) [inline src/library/Uno.Material/Converters/FirstCharacterConverter.cs:13]: Isn't this a breaking change? Are we okay with that?

Comment (Youssef1313) [reply src/library/Uno.Material/Converters/FirstCharacterConverter.cs:13]: I see you're documenting that in #1326.

Though the scale of the breaking change is quite scary. Is this going to be in a major version? I think such break in a minor version will be too much.

Comment (pictos) [reply src/library/Uno.Material/Converters/FirstCharacterConverter.cs:13]: Yes will be a major version. @kazo0 is aware of it and has ever everything planned. If you see, this PR is just a merge from feature to master, the team has reviewed this code before.

**Review by kazo0 — APPROVED** [review]

File: doc/material-migration.md
Comment (kazo0) [inline doc/material-migration.md:8]: Just leaving a comment here to remind us to hold off merging until 5.1 release is out

**Comment by kazo0** [comment]
@pictos should be good to merge once the conflicts are resolved since we just merged in a change to remove the SurfaceTint attached properties :)

**Pull request merged**
